src/app/modules/convai-checker/convai-checker.component.spec.ts

```diff
@@ -1520,25 +1409,19 @@ describe('Convai checker test', () => {
       expectedFeedbackTextVisibilitiesAfterLoading);
   });
```

Comment:
I don't see any asserts or expectations in this test. Is that intentional? Just want to make sure I'm not missing anything :-)